docs/src/components/landing/hero.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { ProjectBadge } from "formidable-oss-badges";
+
+export default function Hero(): JSX.Element {
+  return (
+    <div>
+      <ProjectBadge
+        color="#190E"
```

Comment:
for the sake of the draft - thoughts on running with #37B756 as the green here? it ties to the feature assets that are getting used below and is at least a jumping-off point for continuity.

Reply:
Yea, I'm open to changing the color for sure. Actually, @AlexSwanson pointed out there's a blue color in the Figma designs specifically for react swipeable, so I'm going to build a palette off of that.

<img width="644" alt="image" src="https://github.com/FormidableLabs/react-swipeable/assets/3632381/202e815b-8be9-45fb-8e08-e9d8aedae167">

Reply:
@BrittanyIRL I've updated to use new colors based off of this image. When you get a chance, take a look and let me know what you think. I'm ok with updating or having design provide better assets. I did what I could with Figma, but admittedly it's not my area of expertise.